linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/shared.xml

```diff
@@ -0,0 +1,18 @@
+{{% set smartcard_packages = ['pam_pkcs11', 'mozilla-nss', 'mozilla-nss-tools', 'pcsc-ccid', 'pcsc-lite', 'pcsc-tools', 'opensc', 'coolkey'] %}}
```

Comment:
I think this should be renamed to `linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/sle12.xml` otherwise it will override the content for other products and in this case the platform is including only `sle` products. I have to test this.

Reply:
I guess it should be fine to leave it like this. I've tested a few things it's working as expected for other products.